packages/meditrak-app-server/src/routes/social/SocialFeedRoute.ts

```diff
@@ -115,7 +115,9 @@ export class SocialFeedRoute extends Route<SocialFeedRequest> {
     });
 
     const hasMorePages = feedItems.length > numberPerPage;
-    const items = await Promise.all(feedItems.slice(0, numberPerPage).map(f => f.getData()));
+    const items = (
+      await Promise.all(feedItems.slice(0, numberPerPage).map(f => f.getData()))
+    ).map(item => ({ ...item, creation_date: new Date(item.creation_date).toJSON() }));
```

Comment:
Was this previously broken?

Reply:
Ummm sort of? So the `leaderboard` has always used a `toJSON()` timestamp (which looks like `2020-12-15T00:00:00.000Z`) whereas the ones pulled from the database used a different format (`2020-12-15 00:00:00.000+1100`).

The second format was making the tests hard to write as the result basically depended on the database local timezone. I decided we might as well make them consistent while fixing this. So I went with the `toJSON()` implementation?

Reply:
Oh yep sweet :+1: